lib/xeroizer/models/organisation.rb

```diff
@@ -29,6 +29,8 @@ class Organisation < Base
       string    :registration_number
       string    :timezone
       datetime  :created_date_utc, :api_name => 'CreatedDateUTC'
+      string    :sales_tax_basis
+      string    :sales_tax_period
```

Comment:
I need to add the validate method to Xeroizer's "ActiveRecord" because I want to call an arbitrary method to perform this validation.